components/blocks/LeaderboardInfo/LeaderboardInfo.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import resolveConfig from 'tailwindcss/resolveConfig'
+import tailwindConfig from 'root/tailwind.config'
+import Desktop from './Desktop/Desktop.vue'
+import Mobile from './Mobile/Mobile.vue'
+import type { LeaderboardViewModel } from 'lib/api/data-contracts'
+
+export interface LeaderboardInfoProps {
+  leaderboard: LeaderboardViewModel
+}
+
+// TODO: Remove this. Get from model instead.
+const todoPlatforms = ['PS4', 'PC', 'XboxSeriesX']
+
+// TODO: Implement listeners
+const emit = defineEmits<{
+  (event: 'follow', leaderboardId: number): void
+}>()
+
+const isMobile =
+  window.innerWidth <=
+  parseInt(resolveConfig(tailwindConfig).theme.screens.sm.replace('px', ''), 10)
```

Comment:
Quick note: `parseInt()` default is supposed to be `10` however ya you want to explicitly do it because sometimes it isn't for some reason.

Reply:
Yeah, that's why I do that, because I remember reading about that years ago

Reply:
Oh ya this doesn't change. We want to have the site be responsive, like for myself i may open it up on one monitor in half the size (snapped to half the screen) then oh i need to really look, let me maximize (or vice versa). Mobile first responsive design is the way to go.

Reply:
Should be reactive now